server.js

```diff
@@ -10,40 +10,89 @@ const signature = '@!#$%%^&#$!@#^&***()ROBBY';
 
 //functions to talk to DB
 
-let createUserDb = (username, password, leaning, email) => 
```

Comment:
For the createUserDB function - we will need to pass all values that a user inputted into the database so we will need all of those I believe because if not those vbaraiables in your query will return null or fail.

Reply:
I believe you only need to add the values that are "not null" . however, I just changed it so you add the whole object into the query, and then select the values from the object.  That way we don't have to rewrite inputs in two places.